.spelling

```diff
@@ -404,6 +408,7 @@ sa
 sayin
 schemas
 SDKs
+sds
```

Comment:
Should this be SDS?

Reply:
I think the profile name should be sds, lowercase, so I think we need this

Reply:
@frankbu is correct. The lowercase names were what I found and grabbed for the profiles, which meant for consistency I went with `sds`. And thus added it to _.spelling_.